codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2LocalFlowController.java

```diff
@@ -98,7 +114,6 @@ public int windowSize(Http2Stream stream) {
 
     @Override
     public void incrementWindowSize(ChannelHandlerContext ctx, Http2Stream stream, int delta) throws Http2Exception {
-        checkNotNull(ctx, "ctx");
```

Comment:
Was this intentional? I looked through the call path down to the FrameWriter and I don't see a null check.  Am I missing something?